rbac/management/group/relation_api_dual_write_group_handler.py

```diff
@@ -123,39 +130,65 @@ def _replicate(self):
         except Exception as e:
             raise DualWriteException(e)
 
-    def replicate_added_role(self, role: Role):
-        """Replicate added role."""
+    def extend_relations_to_remove(self, relations_to_remove: list[Relationship]):
+        """Extend relations to remove in replication."""
+        if not self.replication_enabled():
+            return
+        self.group_relations_to_remove.extend(relations_to_remove)
+
+    def replicate_added_roles(self, roles: list[Role], remove_default_access_from: Optional[TenantMapping] = None):
```

Comment:
This removes a type warning (in the definer we pass the django manager which is not a list but is iterable)
```suggestion
    def replicate_added_roles(self, roles: Iterable[Role], remove_default_access_from: Optional[TenantMapping] = None):
```

Reply:
Also, do you want this to replicate?

Two options I think...

Option 1: Change the name for this and remove (e.g. `add_roles` and `remove_roles`) and keep the separate `replicate()` method. In which case, also remove the `replicate_deleted_group()` method since it's redundant with `replicate()`. 

or

Option 2: Keep the names, but make the methods replicate, and remove the separate `replicate()` method. (`replicate_deleted_group()` stays in this case)